cmd/main.go

```diff
@@ -131,14 +131,14 @@ func deployMachineSet() {
 
 	for {
 		glog.Info("Trying to deploy Cluster object")
-		if _, err := v1alphaClient.Clusters("test").Create(cluster); err != nil {
+		if _, err := v1alphaClient.Clusters("default").Create(cluster); err != nil {
 			glog.Infof("Cannot create cluster, retrying in 5 secs: %v", err)
 		} else {
 			glog.Info("Created Cluster object")
 		}
 
 		glog.Info("Trying to deploy MachineSet object")
-		_, err = v1alphaClient.MachineSets("test").Create(machineSet)
+		_, err = v1alphaClient.MachineSets("default").Create(machineSet)
```

Comment:
`default` still better than `test`. Though, is this the final namespace? I would rather make the namespace configurable. Either via configmap or via mao ``--namespace`` or similar option.

Reply:
thanks. Yes, there's separate tickets for addressing dedicated namespaces and service accounts.